faqs/how-do-i-get-involved.md

```diff
@@ -0,0 +1,10 @@
+---
+title: How can I get involved
+weight: 9.1
+attribution:
+  name: sum guy on reddit
+  link: /u/twoofspades
+---
+
+See [Rayonism](https://rayonism.io/) on how to contribute. If your a dapp developer then consider deploying your dapp on on the merge testnet during the ethglobal hackathon (April 16 - May 14). If your a rollup project and would like to see a glimpse of the future? Then make use of sharded data on the merge testnet.
+If your a user then be on the lookout for dapps that deploy on the upcoming testnet. If your a protocol developer and would like to help with the merge process then I suggest asking in the Eth R&D discord.
```

Comment:
- Please replace instances of "your" with "you're"
- "Eth" should be "ETH"
- Please link to ETH R&D Discord via invite link if possible